lint.config.js

```diff
@@ -0,0 +1,5 @@
+const { readFileSync } = require('fs');
+
+module.exports = {
+  logic: JSON.parse(readFileSync('./.eslintrc').toString()),
+};
```

Comment:
I don't think quality gate is being run on the project. There should be a new line at the end of the file.